bpx/__init__.py

```diff
@@ -1,13 +1,102 @@
-"""BPX schema and parsers"""
-# flake8: noqa F401
+from .expression_parser import ExpressionParser
+from .function import Function
+from .interpolated_table import InterpolatedTable
+from .schema import BPX, check_sto_limits
+from .utilities import get_electrode_concentrations, get_electrode_stoichiometries
 
 __version__ = "0.4.0"
 
+__all__ = [
+    "BPX",
+    "ExpressionParser",
+    "Function",
+    "InterpolatedTable",
+    "check_sto_limits",
+    "get_electrode_concentrations",
+    "get_electrode_stoichiometries",
+    "parse_bpx_file",
+    "parse_bpx_obj",
+    "parse_bpx_str",
+]
 
-from .interpolated_table import InterpolatedTable
-from .expression_parser import ExpressionParser
-from .function import Function
-from .validators import check_sto_limits
-from .schema import BPX
-from .parsers import parse_bpx_str, parse_bpx_obj, parse_bpx_file
-from .utilities import get_electrode_stoichiometries, get_electrode_concentrations
+
+def parse_bpx_obj(bpx: dict, v_tol: float = 0.001) -> BPX:
```

Comment:
why have these been moved here from `parsers`?

Reply:
I agree, this should go back to parsers

Reply:
I covered this in #79